storage/src/nodestore/alloc.rs

```diff
@@ -490,19 +542,9 @@ impl<S: WritableStorage> NodeStore<Committed, S> {
             .increment(AREA_SIZES[area_size_index as usize]);
 
         // The area that contained the node is now free.
-        let area: Area<Node, FreeArea> = Area::Free(FreeArea::new(
-            self.header.free_lists()[area_size_index as usize],
-        ));
-
-        let stored_area = StoredArea::new(area_size_index, area);
-
-        let stored_area_bytes = serializer().serialize(&stored_area).map_err(|e| {
-            self.storage.file_io_error(
-                Error::new(ErrorKind::InvalidData, e),
-                addr.get(),
-                Some("delete_node".to_string()),
-            )
-        })?;
+        let mut stored_area_bytes = Vec::with_capacity(16);
```

Comment:
Can we use `std::mem::size_of<T>` here instead of magic 16?

If not, then let's make it a `const` somewhere to call out what 16 represents.

Reply:
16 is not significant. I can make a constant for this and size it correctly. It's really `size_of<u64>() + sizie_of<u8>()`.